sources/logdog/src/log_request.rs

```diff
@@ -9,13 +9,16 @@
 //! these provide the list of log requests that `logdog` will run.
```

Comment:
I can't leave a suggestion on the appropriate lines since they didn't change, but we should update the documentation since it's no longer accurate with the new mode.  I suggest this change:

```diff
--- a/sources/logdog/src/log_request.rs
+++ b/sources/logdog/src/log_request.rs
@@ -33,11 +33,12 @@ pub(crate) fn log_requests() -> Vec<&'static str> {
         .collect()
 }
 
-/// A logdog `LogRequest` is in the format `mode filename instructions`. `mode` specifies what type
-/// of command it is, e.g. `exec ` for a command or `http` for an HTTP get request. `filename` is
-/// the name of the output file. `instructions` is any additional information needed.  For example,
-/// an `exec` request'ss instructions will include the program and program arguments. An `http`
-/// request's instructions will be the URL.
+/// A logdog `LogRequest` represents a line from the config file.  It starts with a "mode" that
+/// specifies what type of request it is, e.g. `exec ` for a command or `http` for an HTTP get
+/// request. Some modes then require a `filename` that determines where the data will be saved in
+/// the output tarball.  The final field is `instructions` which is any additional information
+/// needed by the given mode.  For example, an `exec` requests' instructions will include the
+/// program and program arguments. An `http` request's instructions will be the URL.
 ///
 /// # Examples
 ///
@@ -61,11 +62,18 @@ pub(crate) fn log_requests() -> Vec<&'static str> {
 /// ```text
 /// file some-conf /etc/some/conf
 /// ```
+///
+/// This request will copy files with a known prefix into the tarball; this can be useful for dated
+/// log files, for example.
+///
+/// ```text
+/// glob /var/log/my-app.log*
+/// ```
 #[derive(Debug, Clone)]
 struct LogRequest<'a> {
-    /// The log request mode. For example `exec`, `http`, or `file`.
+    /// The log request mode. For example `exec`, `http`, `file`, or `glob`.
     mode: &'a str,
-    /// The filename that the logs will be written to.
+    /// The filename that the logs will be written to, if appropriate for the mode.
     filename: &'a str,
     /// Any additional instructions or commands needed to fulfill the log request. For example, with
     /// `exec` this will be a program invocation like `echo hello world`. For an `http` request this
```

Reply:
Thanks for the complete write-up. 
I see that you already approved, still fixed in `srgothi92 force-pushed the srgothi92:develop branch from 97a8a4e to 6e258a8 5 minutes ago` :P